octopus_deploy/datadog_checks/octopus_deploy/check.py

```diff
@@ -341,6 +342,29 @@ def _collect_server_nodes_metrics(self):
             self.gauge("server_node.in_maintenance_mode", maintenance_mode, tags=self._base_tags + server_tags)
             self.gauge("server_node.max_concurrent_tasks", max_tasks, tags=self._base_tags + server_tags)
 
+    def _collect_machine_metrics(self, space_id):
+        self.log.debug("Collecting server node metrics.")
+        url = f"api/{space_id}/machines"
+        response_json = self._process_paginated_endpoint(url)
+        machines = response_json.get('Items', [])
+
+        for machine in machines:
+            machine_id = machine.get("Id")
+            machine_name = machine.get("Name")
+            machine_slug = machine.get("Slug")
+            roles = machine.get("Roles", [])
+            health_status = machine.get("HealthStatus", None)
```

Comment:
Question: What are the possible values for this property?

Reply:
Good catch- the four options are:
```
Healthy
Healthy with warnings
Unhealthy
Unavailable
```
so we should check if the status starts with Healthy!

Reply:
Do you think it's better to report this as a metric? I wonder if it wouldn't be better to report it as a tag? What are the benefits of reporting it as a metric?